electrum/gui/qml/qeconfig.py

```diff
@@ -358,13 +366,6 @@ def formatMilliSats(self, amount, with_unit=False):
         else:
```

Comment:
side-note: what's up with this fallback return value?
https://github.com/spesmilo/electrum/blob/d7bc617034377119523709c504edb1205eaae34c/electrum/gui/qml/qeconfig.py#L353-L354